python/lsst/ci/build.py

```diff
@@ -30,13 +30,14 @@ def product_representer(dumper, data):
     return dumper.represent_mapping('tag:yaml.org,2002:map', obj)
 
 
-yaml.add_representer(Product, product_representer)
+yaml.add_representer(models.Product, product_representer)
 
 
 def declare_eups_tag(tag, eups_obj):
-    """ Declare a new EUPS tag
-        FIXME: Not sure if this is the right way to programmatically
-               define and persist a new tag. Ask RHL.
+    """Declare a new EUPS tag.
+
+    FIXME: Not sure if this is the right way to programmatically
+           define and persist a new tag. Ask RHL.
```

Comment:
Please file a ticket for this and reference it here.

Reply:
I've created a generic ticket for additional documentation fixes; https://jira.lsstcorp.org/browse/DM-26180